src/base/utils/version.h

```diff
@@ -177,7 +177,7 @@ namespace Utils
         }
 
         template <typename StringsList>
-        Version(const StringsList &versionParts)
+        constexpr Version(const StringsList &versionParts)
```

Comment:
It still unusable at compile time since QStringList can't be constructed at that time, isn't it?